server/plugin/subscriptions.go

```diff
@@ -178,15 +178,8 @@ func (p *Plugin) SubscribeOrg(ctx context.Context, githubClient *github.Client,
 	return p.Subscribe(ctx, githubClient, userID, org, "", channelID, features, flags)
 }
 
-func (p *Plugin) IsNotificationOff(repoName string) bool {
-	repos, err := p.GetExcludedNotificationRepos()
-	if err != nil {
-		p.API.LogWarn("Failed to check the disabled notification repo list.", "error", err.Error())
-		return false
-	}
-	if len(repos) == 0 {
-		return false
-	}
+func (p *Plugin) IsNotificationOff(repoName string, s *Subscription) bool {
```

Comment:
Can this make be more self explaining?

Reply:
done, let me know if it's still not helpful.